src/cript/nodes/subobjects/identifier.py

```diff
@@ -1,4 +1,42 @@
-class Identifier:
+from typing import Union,  List
+from numbers import Number
+from dataclasses import dataclass, replace
+from cript.nodes.core import BaseNode
+
+class Identifier(BaseNode):
     """
     Identifier subobject
     """
+    @dataclass(frozen=True)
```

Comment:
the identifier node will be obsolete as it has been flattened and merged to the material node